lib/order.rb

```diff
@@ -0,0 +1,20 @@
+class Order
+  attr_reader :basket, :total
+
+  def initialize
+    @basket = Hash.new(0)
+  end
+
+  def add_basket(item, qty)
```

Comment:
Lint/UnusedMethodArgument: Unused method argument - `qty`. If it's necessary, use `_` or `_qty` as an argument name to indicate that it won't be used. (https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars)